discovery-provider/src/api/v1/models/users.py

```diff
@@ -40,6 +40,8 @@
         "repost_count": fields.Integer(required=True),
         "track_count": fields.Integer(required=True),
         "is_deactivated": fields.Boolean(required=True),
+        "erc_wallet": fields.String(requred=True),
+        "spl_wallet": fields.String(required=True),
```

Comment:
you think this would be an issue if solana went down and it failed to make a user bank for this user on creation?
I guess we're switching it all to solana eventually anyways

Reply:
of course an issue, but not a whole lot i think we can do there. good callout though